README.md

```diff
@@ -12,15 +12,39 @@ all the heartbleed.
 
 Conduit is **experimental**. Currently, it _only supports HTTP/2_ and is especially
 well-suited for [gRPC][grpc]. Follow our progress towards production-readiness here and on
-[twitter][twitter].
+[Twitter][twitter].
 
 <!-- TODO add roadmap link -->
 
+## Documentation
+
+View [Conduit docs][conduit-docs] for more a more comprehensive guide to
+getting started.
+
+
+## Getting started with Conduit
+
+1. Install the Conduit CLI `curl https://run.conduit.io/install | sh `
+
+2. Copy `/Users/username/.conduit/bin/conduit` into your PATH
+
+3. Run `conduit install | kubectl apply -f -` to install Conduit
+
+4. Check to make sure you're running the latest version _(0.1.0)_ of Conduit
+with `conduit version`
+
+5. Open the local version the Conduit controller with `conduit dashboard`
```

Comment:
Typo in this sentence. Maybe: Open _a_ local version _of_ the Conduit controller....